.github/workflows/_win-test.yml

```diff
@@ -93,9 +105,8 @@ jobs:
         id: monitor-script
         shell: bash
         run: |
-          python3 -m pip install psutil==5.9.1
-          python3 -m pip install pynvml==11.4.1
-          python3 -m tools.stats.monitor > usage_log.txt 2>&1 &
+          # Windows conda doesn't have python3 binary, only python, but it's python3
```

Comment:
Just to clarify, both `python` and `python3` commands would work here, right? Otherwise the old version of the could would be broken.

Reply:
I think so, both command works.  So the story here is that we can either get Python binary from setup-python GitHub action (the old way which require network or caching), or from Windows conda pre-installed in the AMI (the new way proposed by this PR which doesn't require any network traffic).  Due to the way setup-python is implement, it provides a python3 binary, which conda just calls it python